.github/workflows/component-builder-publish.yml

```
@@ -1,61 +1,93 @@
+            podman pull ${S390_SOURCE_IMG}
+
+            # Tag the images
+            podman tag ${X86_64_SOURCE_IMG} ${X86_64_DEST_IMG}
+            podman tag ${AARCH64_SOURCE_IMG} ${AARCH64_DEST_IMG}
+            podman tag ${S390_SOURCE_IMG} ${S390_DEST_IMG}
+
+            # Push the images
+            podman push ${X86_64_DEST_IMG}
+            podman push ${AARCH64_DEST_IMG}
+            podman push ${S390_DEST_IMG}
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)